packages/rest/src/router/router-base.ts

```diff
@@ -18,12 +18,29 @@ export abstract class BaseRouter implements RestRouter {
    */
   protected routesWithoutPathVars: {[path: string]: RouteEntry} = {};
 
-  protected getKeyForRoute(route: RouteEntry) {
-    const path = route.path.startsWith('/') ? route.path : `/${route.path}`;
-    const verb = route.verb.toLowerCase() || 'get';
+  /**
+   * Build a key for verb+path
+   * @param verb HTTP verb/method
+   * @param path URL path
+   */
+  private getKey(verb: string, path: string) {
```

Comment:
Since this method is not using anything from the class instance (no `this`), can you move it outside of the class into a regular function?